### ISSUE TITLE
detecting all 0 when model diverges

### ISSUE THREAD

**Issue by Pari-singh** [zzh8829/yolov3-tf2#45]
I don't understand if the model should just output 0 for everything even if the model diverges a little bit, i.e., I get the output only for 2 epochs where train and val loss are almost same, but even for 1 epoch later I get all 0. Is this a expected behaviour or I am doing something wrong?

Does someone happen to know why this is the case?

**Comment by zzh8829** [comment]
it is expected since trianing yolov3 is very difficult, please see the updated training tutorial
